tests/test_winogender_metric.py

```diff
@@ -0,0 +1,58 @@
+import csv
+import os
+import shutil
+from string import ascii_lowercase
+import tempfile
+import unittest
+import pandas as pd
+
+from src.metrics.winogender_metrics import GenderParity
+
+
+class TestWinogenderMetric(unittest.TestCase):
+    def setUp(self):
+        self.temp_dir = tempfile.mkdtemp()
+        self.path = os.path.join(self.temp_dir, "temp_ccg_dataset.tsv")
```

Comment:
why ccg?

Reply:
Oh this actually isn't used and is copy-pasta, deleting.